### ISSUE TITLE
Heroku demo version error

### ISSUE THREAD

**Issue by mredigonda** [digitallysavvy/group-video-chat#13]
Hi @digitallysavvy, I stumbled upon a demo of this code (probably an outdated version) running at https://agora-group-video-chat.herokuapp.com/

That version only asks the person for "Agora AppId" and "Channel", but after you pass those it returns an error "17:25:08:788 Agora-SDK [ERROR]: [BE367] Get server node failed [DYNAMIC_USE_STATIC_KEY] https://webrtc2-ap-web-1.agora.io/api/v1 DYNAMIC_USE_STATIC_KEY"

![image](https://user-images.githubusercontent.com/25920622/102539796-e9f9e680-408c-11eb-9e40-75bf3c3da465.png)

Maybe you actually don't expect people to use that Heroku demo, but this one instead: https://digitallysavvy.github.io/group-video-chat/

The thing is that the Heroku demo is pointed to from the Medium guide you created: https://medium.com/agora-io/agora-cloud-recording-quickstart-guide-with-postman-demo-c4a6b824e708

Maybe you want to update the link and turn off that Heroku app, I just wanted to let you know this is happening.

Best regards.

**Comment by digitallysavvy** [comment]
@mredigonda thanks for bringing this to my attention.

You are correct the Heroku version is very old. The latest demo version is the GitHub.io link (https://digitallysavvy.github.io/group-video-chat/). The issue you are facing is the appId that you are using has tokens enabled and the demo on Heroku did not have that update, so its not as secure and will throw an error when trying to join the channel without the token.

I updated the Medium post for the Postman demo.

**Comment by mredigonda** [comment]
Great, I'm glad that helped.

I'm also a bit worried about this bit of the documentation: https://docs.agora.io/en/cloud-recording/cloud_recording_rest?platform=RESTful&changePlatformAlert=RESTful#a-namedemo-restasample-code (the sample Python code).

I tried the sample code there in Python and couldn't get the Cloud Recording to work properly. I will describe how I filled the data:
- APPID: this one is easy to get
- Auth: I used a Python snippet for this one: `"Basic " + base64.b64encode( (CUSTOMERID + ":" + CUSTOMERCERTIFICATE).encode('ascii') ).decode('ascii')`
- Cname: I used "hello" as channel name, and used it consistently
- ACCESS_KEY, SECRET_KEY, VENDOR, REGION: I know these are set up correctly because I already used the exact same values to upload files to my S3 bucket from another script, and it worked
- BUCKET: This is the only one I'm not sure if I filled properly, but I tried with normal name "my-experiments", the ARN (Amazon Resource Name) and even the URL

I also already checked [common errors like the 404-after-start FAQ](https://docs.agora.io/en/All/faq/return-404), and turning on Cloud Recording from the Agora.io dashboard.

This all leads me to believe that Python script is also somehow outdated. What are your thoughts on this?

Sorry if this is not specifically related to this repository, ideally it should be a mail but I couldn't find yours.

Thanks a lot.
